src/features/annotation-thread/AnnotationThread.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @flow
+ * @file Annotation Thread Container
+ * @author Box
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import AnnotationThreadContent from './AnnotationThreadContent';
+import API from '../../api/APIFactory';
+import APIContext from '../../elements/common/api-context';
```

Comment:
It's not ideal for a feature-level component to import modules from the `elements` directory. It doesn't seem like APIContext or Internationalize add a ton of value here. Can we pass the API instance down directly as a prop and use `IntlProvider`, instead?